srcpkgs/python3-cssselect2/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-cssselect2'
+pkgname=python3-cssselect2
+version=0.6.0
+revision=1
+wrksrc="cssselect2-${version}"
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-flit_core python3-webencodings python3-tinycss2"
+depends="python python3-flit_core"
```

Comment:
python2? flit_core is usually only used for build. webencodings and tinycss2 is probably missing